TypeCobol.LanguageServer/TypeCobolCustomLanguageServer/TypeCobolCustomLanguageServer.cs

```diff
@@ -171,7 +171,13 @@ protected virtual void OnDidReceiveExtractUseCopies(ExtractUseCopiesParams param
                 {
                     var missingCopiesParam = new MissingCopiesParams();
                     missingCopiesParam.textDocument = parameter.textDocument;
+#if EUROINFO_RULES
+                    missingCopiesParam.Copies = copiesName.Where(s => !base.Workspace.CompilationProject.CompilationOptions.HasCpyCopy(s)).ToList();
+                    missingCopiesParam.cpyCopies = copiesName.Where(s => base.Workspace.CompilationProject.CompilationOptions.HasCpyCopy(s)).ToList();
+#else                    
                     missingCopiesParam.Copies = copiesName;
+                    missingCopiesParam.cpyCopies = new List<string>();
+#endif
```

Comment:
Can you factorize this into a single method to avoid duplication with `TypeCobolServer` ?
Some other remarks:
- you can use LINQ `ToLookup` method to split copies names into 2 groups in a single call, instead of iterating the list twice
- no need to use the `base` keyword as the `Workspace` property is not redefined

Reply:
done in 1cf225e